src/middleware/authMiddleware.js

```diff
@@ -0,0 +1,35 @@
+import ClientOAuth2 from 'client-oauth2';
+
+// TODO: move values to env variables
+export const hsesAuth = new ClientOAuth2({
+  clientId: '7b9268101989e6ce7efc',
+  clientSecret: 'ABHnjtF6iB1fsg940xBHvZy89Las7X31gywtsMChHrMV',
```

Comment:
Should these be passed in as env variables?

Reply:
Yes, that's the plan. There is a 'TODO' on line 3.

Reply:
Oops missed that!